### PR TITLE
Descriptor form

### PR DESCRIPTION
**Pull request by dzhang55**
- Added form for adding a descriptor and editting a descriptor
- Added views for managing existing descriptors, removing descriptors, adding new descriptors

Note: Currently the buttons aren't spaced nicely, and the icons for some of them are incorrect

### CODE REVIEW

**Comment by weronica** [comment]
Could you post a screenshot of your UI changes?

**Comment by dzhang55** [comment]
screen shot 2016-03-26 at 9 32 07 am
screen shot 2016-03-26 at 9 32 23 am

**Comment by dzhang55** [comment]
screen shot 2016-03-26 at 9 33 34 am
(incorrect icons)

**Comment by armant** [comment]
r @weronica

File: app/admin/views.py [hunk 1]
```diff
@@ -4,13 +4,15 @@
...
     ChangeAccountTypeForm,
+    ChangeUserEmailForm,
+    EditDescriptorForm,
     InviteUserForm,
+    NewDescriptorForm,
+    NewUserForm
 )
 from . import admin
-from ..models import User, Role
+from ..models import User, Role, Descriptor
```
Comment (weronica) [inline app/admin/views.py]: nit: alphabetical ordering

File: app/admin/views.py [hunk 2]
```diff
@@ -159,3 +150,75 @@ def delete_user(user_id):
...
+    """Create a new descriptor."""
+    form = NewDescriptorForm()
+    for i in range(10):
+        form.option_values.append_entry()
+    if form.validate_on_submit():
+        values = []
+        for v in form.option_values.data:
+            if v is not None and len(v) != 0:
+                values.append(v)
+        print values
```
Comment (weronica) [inline app/admin/views.py]: nit: remove print statement

File: app/descriptor/views.py [hunk 3]
```diff
@@ -1,8 +1,86 @@
-from flask import render_template
+from flask import render_template, redirect, flash, url_for
```
Comment (weronica) [inline app/descriptor/views.py]: nit: alphabetical order

**Pull request merged**